modules/Notes/Note.php

```diff
@@ -205,6 +205,24 @@ public function get_summary_text()
         return "$this->name";
     }
 
+    /**
+     * Returns the content or an empty string if there is no attachment or it
+     * couldn't be located.
+     *
+     * @return string
+     */
+    public function getAttachmentContent()
```

Comment:
This function returns an empty string in cases:
- file not exists
- file_get_contents() failure
- file is empty
I think it's confusing, the caller does not know why is the empty string passed if any problem occured.

Reply:
I've changed it to return false or a string now.